src/artifacts/FalconHeavyWithLogos.tsx

```diff
@@ -108,4 +108,4 @@ export function FalconHeavyWithLogos({
   );
 }
 
-useGLTF.preload('/assets/models/falcon-heavy-with-logos-transformed.glb');
+//useGLTF.preload('/assets/models/falcon-heavy-with-logos-transformed.glb');
```

Comment:
Did you mean to leave these commented out?

Reply:
Indeed, we must find a solution to retain the `preload` function with the accurate GLB file path in each component file while ensuring the successful execution of tests. Uncommenting it currently leads to the failure of all tests associated with the 3D component.

Reply:
@ajd6735 can you uncomment these so that we can see the test failure details? I don't think we'd want to merge this with reduced functionality.

Reply:
Honestly, I had a similar thought. We require the `preload` function for asynchronous rendering, and it's crucial to ensure the project builds successfully, even if the 3D components are not thoroughly tested.

Reply:
I can't uncomment the preload function because the tests fail and I can't commit. 
<img width="663" alt="Screenshot 2023-12-05 234503" src="https://github.com/MissionToPsyche/tungsten_01b_ar_xr-psb/assets/117790810/87d7e0e6-2661-43e7-ae1d-8a2a0f9afa33">